config_templates/gretel/synthetics/differential-privacy.yml

```diff
@@ -15,17 +15,18 @@ models:
   - synthetics:
       data_source: __tmp__
       params:
-        epochs: 50
-        vocab_size: 0
-        learning_rate: .001
-        rnn_units: 256
-        batch_size: 4 
+        epochs: auto
+        vocab_size: auto
+        learning_rate: auto
+        rnn_units: auto
+        batch_size: auto
         predict_batch_size: 1
         dp: True
         dp_noise_multiplier: 0.001
         dp_l2_norm_clip: 5.0
         dp_microbatches: 1
+      generate:
+        num_records: 5000
       privacy_filters:
-        similarity: "high"
-        outliers: "high"
-        
\ No newline at end of file
+        similarity: auto
```

Comment:
Not sure where we left this - should we set these to null explicitly based on @lipikaramaswamy's feedback? Or add logging events showing that they have been turned off?

Reply:
@sami-t will that result in the PPL being set to "Normal" when it should really be "Excellent"?

Reply:
Good question, no, turning DP on is sufficient to get an excellent score regardless of privacy filters